### PR TITLE
refactor!: Rename path to secret name

### PR DESCRIPTION
**Pull request by vli11**
BREAKING CHANGE: Rename path to secret name

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

## Testing Instructions
make test
make docker
/compose-builder$ make gen no-secty ds-onvif-camera (this generates docker-compose.yml with latest edgex images)
Replace onvif docker image in docker-compose.yml with your latest image created using make docker from the previous step
/compose-builder$ make up
/bin/configure-subnets.sh without error
/bin/map-credentials.sh without error
no error in device-onvif-camera service log

### CODE REVIEW

File: doc/credentials.md [hunk 1]
```diff
@@ -59,10 +59,10 @@ See [here](./utility-scripts.md) for the full guide.
 > **Note:** Replace `<secret-name>` with the name of the secret, `<username>` with the username,
 > `<password>` with the password, and `<mode>` with the auth mode.
 
-Set Path to `<device-name>`
+Set SecretName to `<device-name>`
```
Comment (lenny-goodell) [inline doc/credentials.md:62]: @ajcasagrande , what is this trying to say? Should it be:
```suggestion
Set Secret for `<device-name>`
```

Comment (lenny-goodell) [reply doc/credentials.md:62]: @ajcasagrande confirm @vli11 change is correct.

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Review by presatish — APPROVED** [review]

**Comment by presatish** [comment]
@vli11 the testing instructions provided seem to be incorrect. You can refer https://github.com/edgexfoundry/device-onvif-camera/pull/254 for the right one.

**Comment by vli11** [comment]
> @vli11 the testing instructions provided seem to be incorrect. You can refer #254 for the right one.

thanks @presatish for sharing the test instruction, I have updated again and run the test successfully. The only test instruction difference from #254 is I don't have a real onvif camera, so I can't see device been registered, but I make sure no errors in the log.

**Pull request merged**